docs/calculations.mdx

```diff
@@ -367,20 +377,26 @@ creative_data_transfer_usage_emissions_gco2pm =
     lookup_carbon_intensity_gco2e_per_kwh(country, region, utc_datetime)
 
 power_watts =
-    baseload_watts_per +
-    dynamic_watts_per_kbps x default_video_bitrate[device_type]
+    baseload_watts +
+    dynamic_watts_per_mbps x default_video_bitrate_kbps[device_type] / 1000
 
 creative_data_transfer_embodied_emissions_gco2pm =
       creative_video_duration_seconds x
-      default_video_bitrate[device_type] / 128 x
+      default_video_bitrate_kbps[device_type] / 128 x
       embodied_emissions_gco2e_per_kb
 ```
 
 ### Creative Delivery - Platform
 
 ```
+default_creative_platforms =
+    ad_format.is_3p_served ?
+    [generic_creative_ad_server, generic_measurement_platform] :
+    []
+creative_platforms = creative_ad_platforms ?? default_creative_platforms
```

Comment:
Should this be `creative.ad_platforms ?? default_creative_platforms` Assuming the first one is if the ad platforms have been given on the creative input?

Reply:
technically it would be `delivery_row.creative_ad_platforms` - I haven't been adding the delivery row part but we could change all of them?